sklearn/feature_extraction/text.py

```diff
@@ -479,6 +479,44 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
     dtype : type, optional
         Type of the matrix returned by fit_transform() or transform().
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> corpus = [
+    ...     'This is the first document.',
+    ...     'This is the second second document.',
+    ...     'And the third one.',
+    ...     'Is this the first document?',
+    ... ]
+    >>> vectorizer = HashingVectorizer()
+    >>> print(vectorizer._get_hasher())  # doctest: +NORMALIZE_WHITESPACE
+    FeatureHasher(alternate_sign=True, dtype=<class 'numpy.float64'>,
+           input_type='string', n_features=1048576, non_negative=False)
+    >>> X = vectorizer.fit_transform(corpus)
+    >>> print(X.toarray())  # doctest: +NORMALIZE_WHITESPACE
+    [[ 0.  0.  0. ...,  0.  0.  0.]
+     [ 0.  0.  0. ...,  0.  0.  0.]
+     [ 0.  0.  0. ...,  0.  0.  0.]
+     [ 0.  0.  0. ...,  0.  0.  0.]]
+    >>> print(X.shape)
+    (4, 1048576)
+    >>> vectorizer = HashingVectorizer(n_features=2**3)
```

Comment:
There're actually two examples here. We aim to provide users with a minimal example and your examples seems too long from my side. Maybe remove the previous example, unless you have some specific reason?

Reply:
I'll remove the first example with default `n_features=2**20`, plus shorten it.